sound/soc/sof/loader.c

```diff
@@ -192,6 +192,42 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				   const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+	int elems_counter;
+	int elems_size;
+	int i;
+
+	/* calculate elements counter */
+	elems_size = config->hdr.size - sizeof(struct sof_ext_man_elem_header);
+	elems_counter = elems_size / sizeof(struct sof_config_elem);
+
+	dev_info(sdev->dev, "ext_man_get_config_data can hold up to %d config elements\n",
+		 elems_counter);
+
+	for (i = 0; i < elems_counter; ++i) {
+		dev_info(sdev->dev, "ext_man_get_config_data get index %d token %d val %d\n",
```

Comment:
I think `dev_dbg()` should be enough for this one, possibly even for the one above

Reply:
Sure, I forget about it